test/nodeMock/reqResMapping.js

```diff
@@ -63,5 +87,97 @@ function getJurisdictions(){
 
 
 
-module.exports = { requestMapping,configurations};
+const createCase = () => {
+    return {
+        caseCreatedDate: faker.date.past(),
+        caseDueDate: faker.date.future(),
+        caseRef: faker.random.uuid(),
+        caseType: `CaseType${caseTypeNumber}`,
+        petFirstName: faker.name.firstName(),
+        petLastName: faker.name.lastName(),
+        respFirstName: faker.name.firstName(),
+        respLastName: faker.name.lastName(),
+        sRef: faker.random.uuid(),
+    }
+}
 
+const createCaseData = (numUsers = 5) => {
+    return new Array(numUsers)
+        .fill(undefined)
+        .map(createCase)
+}
+
+let caseTypeNumber = 0;
```

Comment:
Again I believe this can be done without having global file variables

Reply:
<-- Uday to look at .